android/app/build.gradle

```diff
@@ -23,6 +23,7 @@ if (flutterVersionName == null) {
 
 apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
+apply plugin: 'com.google.gms.google-services'
```

Comment:
This is an android specific plugin needed for the android app which is why we're adding dependencies into the android config files